content/articles/domains-se.md

```diff
@@ -0,0 +1,24 @@
+---
+title: .SE Domains
+excerpt: This article explains the requirements and special procedures for .SE domain names.
+categories:
+- Domains
+---
+
+# .SE Domain Names
+
+* TOC
+{:toc}
+
+---
+
+This article explains the requirements and special procedures for .SE domain names.
+
+
+## Registering a .SE domain {#registering}
+
+### ID Number
+To register a .SE domain, an ID number (e.g. civic registration number, company registration number, ID card/passport number, or the equivalent) is required by NIC-SE. For Swedish citizens, you can use the social security number (Personnummer).
+
+### VAT ID
+To register a .SE domain, a VAT ID has to be stated for companies that are located inside the European Union but outside Sweden. This attribute is optional.
```

Comment:
Does wording it like this make it less confusing?

```suggestion
To register a .SE domain, a VAT ID has to be stated for companies that are located inside the European Union but outside Sweden. This attribute is optional for companies inside Sweden.
```

Reply:
I think so, I updated it to reflect this — thanks @kntsoriano 🙌